shared-bin/src/to_sse_event.rs

```diff
@@ -90,3 +73,28 @@ impl ToSseEvent for Option<Quote> {
         Event::json(self).event("quote")
     }
 }
+
+#[derive(Debug, Clone, Serialize)]
+pub struct MakerCompatibility {
```

Comment:
It's not so easy to move it to a taker specific place (outside of this crate), because you run into not being able to implement a foreign trait on a foreign type.